docker-compose.production.yml

```diff
@@ -191,6 +243,8 @@ services:
       - webnet
       - dbnet
 
+# secrets mounted to /run/secrets/
+# e.g. /run/secrets/petabox_seed
 secrets:
     petabox_seed:
       file: /opt/.petabox/seed
```

Comment:
Should remove this now; no longer need it. Anything that references needs to switch to using the line we added to the nginx.conf for web_nginx.

Reply:
Fixed